_docs/yasnippets/README.md

```diff
@@ -0,0 +1,21 @@
+# Yasnippets Collection
+This folder contains a yasnippets collection with common templates to ease the development. Yasnippets can be integrated into Emacs: https://github.com/joaotavora/yasnippet .
+
+The snippets can also be used to provide coding guidelines and ensure a general look and feel.
+
+## Quickstart
+Copy the folders in this directory into your local yasnippets emacs configuration.
+
+If you use spacemacs they have to be placed here:
+* ` ~/.emacs.d/private/snippets`.
+
+## Usage
+Use the `key` provided in the snippet and let yasnippet expand it.
+Note: Since yasnippet uses the actual mode of the file which should be expanded ensure the mode and the snippets folder name match.
+
+### Example: Makefile Generation
+This requires the `makefile-bsdmake-mode`.
+1. Create `Makefile`
```

Comment:
MD032/blanks-around-lists: Lists should be surrounded by blank

Reply:
Solved with: 98676aecedb3d6a9820c444b650f7910a8ff9dea.